python-backend/app/api/mines/reports/models/mine_report.py

```diff
@@ -56,7 +59,7 @@ def create(cls,
     def find_all_by_mine_guid(cls, _id):
```

Comment:
Can you remove `all` from the prefix in this method, now that it filters on deleted_ind (as it should). thanks. Not a blocker

Reply:
No.